backend/overstroomik_service/geoserver.py

```diff
@@ -0,0 +1,152 @@
+"""
+This class uses the geoserver-api and geoserver layers to find
+location information by rd coordinates
+"""
+
+from typing import Optional
+import httpx
+
+from overstroomik_service.auto_models import Data
+from overstroomik_service.config import settings
+
+ERROR_GENERAL_NOER = "no-error"
+ERROR_GEOS_NO_RESP = "Geen response geoserver"
+ERROR_GEOS_NO_SMAP = "Geen response specifieke kaart"
+
+
+class Geoserver():
+
+    async def get_data(self,
+                       rd_x: float,
+                       rd_y: float,
+                       geoserver_url: Optional[str] = settings.GEOSERVER_URL,
```

Comment:
`Optional` means it can be None, but the provided default always makes this a string. Do we pass `None` ourselves? Otherwise we can just make it a `str`.

Reply:
Optional  is removed